pkg/client/client.go

```diff
@@ -262,7 +262,7 @@ func WithKeychain(keychain authn.Keychain) Option {
 	}
 }
 
-const DockerAPIVersion = "1.38"
+const DockerAPIVersion = "1.44"
```

Comment:
Lifecycle uses `client.WithAPIVersionNegotiation()` - maybe that's safer? https://github.com/buildpacks/lifecycle/blob/73739826a85bc909937c87f0feb3c23e3667c9b5/priv/docker.go#L27

Reply:
Natalie! Yeah, we did that [here](https://github.com/buildpacks/pack/pull/2474)